Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentProducer/IncidentProducer.cs

```diff
@@ -0,0 +1,252 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using StackExchange.Redis;
+using System.Collections.Generic;
+using System.Collections;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Net;
+using System.Text.Json;
+using System.Text;
+using System.Threading.Tasks;
+using System;
+using System.Dynamic;
+using Newtonsoft.Json.Converters;
+using Microsoft.WindowsAzure.Storage;
+using Helios2Sentinel;
+
+namespace Helios2Sentinel
+{
+    public class IncidentProducer
+    {
+        private static readonly object queueLock = new object();
+        private static Lazy<ConnectionMultiplexer> lazyConnection = CreateConnection();
+        private static string containerName = Environment.GetEnvironmentVariable("containerName");
+        private static string blobStorageConnectionString = Environment.GetEnvironmentVariable("BlobStorageConnectionString");
```

Comment:
Why do you need a separate parameter? Reuse AzureWebJobsStorage as it connects to the same storage.

Reply:
I am not sure how to make this change....

Reply:
This 
`private` static string blobStorageConnectionString = GetSecret("AzureWebJobsStorage");

The queue and blob are in the same storage account.